python/tests/core/constraints/test_metric_constraints.py

```diff
@@ -73,7 +73,7 @@ def metric_resolver(profile_view) -> List[Metric]:
 
     legs_less_than_12_constraint = MetricConstraint(
```

Comment:
I think this is good for a temporary fix, but for the future, if it's a `nan`, shouldn't this raise an error when building the metric?

Reply:
This test is an arbitrary example of a metric constraint, I think we can follow up and remove the possibility of seeing `nan` in this metric, but I'm scoping that out of this fix.